docs/usage/configuration-options.md

```diff
@@ -3128,6 +3128,7 @@ This setting tells Renovate when you would like it to raise PRs:
 - `immediate` (default): Renovate will create PRs immediately after creating the corresponding branch
 - `not-pending`: Renovate will wait until status checks have completed (passed or failed) before raising the PR
 - `status-success`: Renovate won't raise PRs unless tests pass
+- `approval`: Renovate waits until getting approval for the update via the Dependency Dashboard
 
 Renovate defaults to `immediate` but you might want to change this to `not-pending` instead.
```

Comment:
Do you think we should warn users to only use _one_ of these_:

- `dependencyDashboardApproval=true`, or
- `prCreation=approval`

Its probably pretty confusing when you mix-and-match options. 😄

Reply:
Yes let's warn them here that it's not compatible with dependencyDashboardApproval=true